### DISCUSSION TITLE
Copilot with Student?

### DISCUSSION THREAD

**Discussion by CarmentaGit** [community/community#38103] [GitHub Education]
Hi!

So I just activated my student account and did everything asked. When I try to access copilot from Settings, it just gives me 60 days try option. I heard that you need to be verified as a student but I can't be sure if I'm verified or not, I have my Pro badge though? Can anybody help with this please, I really wanna try and use that feature.

**Comment by FahdSeddik** [comment]
Hey @Carmentas ,
You should give it time to update. A few days should be enough. Hope that helps!
